core/uemis-downloader.c

```diff
@@ -1161,7 +1202,7 @@ static bool get_matching_dive(int idx, char *newmax, int *uemis_mem_status, devi
 	while (!found) {
 		if (import_thread_cancelled)
 			break;
-		snprintf(dive_to_read_buf, sizeof(dive_to_read_buf), "%d", dive_to_read);
+        snprintf(dive_to_read_buf, sizeof(dive_to_read_buf), "%d", dive_to_read+1);
```

Comment:
Will this work for a new DC? I.e. is the first dive on a device going to be read on a device where the owner has not been reset?

Reply:
Probably not. I will rework this bugfix so it won't create new bugs.